postscripts/check_keytab

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+###
+# Description: Test that a (host) principal in the node's keytab is functional.
+#
+# Notes:
+#
+# 1. Does NOT know if a keytab should be present or not. Assumes
+#    the restore process was successful. If the default keytab is missing
+#    does NOT error; if a (custom) keytab is specified via the -k param
+#    and is missing the script WILL error.
+#
+# 2. Tests to make sure that kinit via the (default or specified) principal is possible.
+#
+# Requirement: Define alertemail in the site table. This could be
+#              an email address that sends to the appropriate alerts
+#              channel in Slack.
+#
+# Usage:
+#
+# - manual: updatenode <noderange> -P "custom/check_keytab"
+#
+# - postscript: include "custom/check_keytab"
+#
+# Source: https://github.com/ncsa/xcat-tools
+###
+
+#
+# PREP
+#
+PRG=$( basename $0 )
+logger -t xcat -p local4.info "running '$PRG' on node $NODE"
+
+
+#
+# DEFAULT SETTINGS
+#
+DEBUG=1
+PRINCIPAL="host/"`hostname -f`"@NCSA.EDU"
```

Comment:
I am worried that there may be cases where `hostname -f` intentionally doesn't match the hostname of the host's kerberos principal. I can't come up with an example at the moment, but I guess that is the point of optionally being able to pass in the host principal name.

No change needed. Just thinking out loud.